webapp/channels/src/components/advanced_text_editor/help_button/help_button.scss

```
@@ -0,0 +1,28 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+.HelpButton {
+    display: flex;
+    justify-content: flex-end;
+    padding-right: 4px;
+
+    &__link {
+        padding: 2px 4px;
+        border: none;
+        background: none;
+        color: rgba(var(--center-channel-color-rgb), 0.56);
+        cursor: pointer;
+        font-size: 11px;
```

My mistake on making this 11px. It should match what the 'user is typing' message is, which appears to be 12px
```suggestion
        font-size: 12px;
```